dcos/mesos.py

```diff
@@ -100,6 +100,52 @@ def get_state_summary(self):
         url = self.master_url('master/state-summary')
         return http.get(url).json()
 
+    def slave_file_read(self, slave_id, params):
```

Comment:
I know this is not part of this PR so okay. But the argument `param` is too obscure to me as user of this library/module. It requires that we document what keys that are allowed in params. Looking at the rest of the code I see that they are `path`, `length`, `offset`. Why not elevate them to the function? E.g. `def slave_file_read(self, slave_id, path, offset, length)`